drivers/i2c/i2c_nrfx_twi_rtio.c

```diff
@@ -32,9 +32,9 @@ BUILD_ASSERT(
 	offsetof(struct i2c_nrfx_twi_common_data, dev_config)
 );
 
-static void i2c_nrfx_twi_complete(const struct device *dev, int status);
+static void i2c_nrfx_twi_rtio_complete(const struct device *dev, int status);
 
-static bool i2c_nrfx_twi_msg_start(const struct device *dev, uint8_t flags,
+static bool i2c_nrfx_twi_rtio_msg_start(const struct device *dev, uint8_t flags,
 				   uint8_t *buf, size_t buf_len, uint16_t i2c_addr)
```

Comment:
Just nitpicking: the indentation here is a bit a bit weird now (also in a few more places below).

Reply:
Thanks, I've added another commit to address non-related indentation changes. LMK if I missed anything